Source/WebCore/dom/Element.cpp

```diff
@@ -2184,6 +2185,24 @@ bool Element::isElementsArrayReflectionAttribute(const QualifiedName& name)
     return false;
 }
 
+void Element::setUserInfo(JSC::JSGlobalObject& globalObject, JSC::JSValue userInfo)
+{
+    auto throwScope = DECLARE_THROW_SCOPE(globalObject.vm());
+
+    auto serializedData = JSONStringify(&globalObject, userInfo, 0);
+    if (throwScope.exception())
+        return;
+
+    ensureElementRareData().setUserInfo(serializedData);
```

Comment:
Could we change the setter to use move semantics?

```suggestion
    ensureElementRareData().setUserInfo(WTFMove(serializedData));
```

Reply:
Done.